examples/inventory-public-testnet.yml

```
@@ -3,7 +3,7 @@
 all:
   vars:
     ansible_user: root
-    chain_version: v9.0.3
+    chain_version: v9.1.0
```

Can we leave this one at v9.0.3?
```suggestion
    chain_version: v9.0.3
```